gbpservice/neutron/tests/unit/nfp/configurator/agents/test_generic_config.py

```diff
@@ -331,22 +326,10 @@ def test_configure_hm_forever_genericconfigeventhandler(self):
         """
 
         ev = fo.FakeEventGenericConfig()
-        ev.data['resource_data'].update({'periodicity': gen_cfg_const.FOREVER})
+        ev.data['resource_data'].update({'periodicity': const.FOREVER})
         ev.id = 'CONFIGURE_HEALTHMONITOR forever'
         self._test_handle_event(ev)
 
-    def test_clear_hm_genericconfigeventhandler(self):
-        """ Implements test case for clear health monitor method
-        of generic config event handler.
-
```

Comment:
This was removed as it was just a placeholder with the child function having no definition. Anyways, I have written a test case for this.